ebr_trackerbot/storage/db.py

```diff
@@ -0,0 +1,135 @@
+"""
+Slack Bot Generic DB Storage
+"""
+
+import logging
+import pendulum
+import sys
+
+
+def create_table(get_connection):
+    """
+    Create table if it does not exist
+    """
+    conn = get_connection()
+    try:
+        cursor = conn.cursor()
+        cursor.execute(
+            "CREATE TABLE IF NOT EXISTS tracks "
+            + "(id integer PRIMARY KEY, user text NOT NULL, test text NOT NULL, "
+            + " expiry timestamp NOT NULL, channel_id text NOT NULL, thread_ts text NOT NULL);"
+        )
+        conn.commit()
+    except:
+        logging.error(str(sys.exc_info()[1]))
+        conn.close()
```

Comment:
Close should be called in my opinion also in case the execution succeeded

Reply:
Why it should re-connect to database after every query?

Reply:
@martinm82 I think instead that close should not be called here at all, as the connection is not actually created here.

Reply:
It was rewritten with "with" statement